keyboards/cannonkeys/ortho48/keymaps/BasicOrtho48/keyDefinitions.h

```diff
@@ -0,0 +1,50 @@
+
+//Shorthand
+
+//F Keys
+#define F_1 KC_F1
```

Comment:
These seem kinda pointless. We already have "shorthand" versions of most keycodes eg. `KC_P1` for `KC_KP_1`.